flickit-advice-engine/src/test/java/org/flickit/assessment/advice/application/port/in/advicenarration/CreateAssessorAdviceNarrationUseCaseParamTest.java

```diff
@@ -0,0 +1,43 @@
+package org.flickit.assessment.advice.application.port.in.advicenarration;
+
+import jakarta.validation.ConstraintViolationException;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.flickit.assessment.advice.common.ErrorMessageKey.CREATE_ASSESSOR_ADVICE_NARRATION_ASSESSMENT_ID_NOT_NULL;
+import static org.flickit.assessment.advice.common.ErrorMessageKey.CREATE_ASSESSOR_ADVICE_NARRATION_ASSESSOR_NARRATION_SIZE_MAX;
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_ID_NOT_NULL;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CreateAssessorAdviceNarrationUseCaseParamTest {
+
+    @Test
+    void testCreateAssessorAdviceNarration_assessmentIdIsNull_ErrorMessage() {
```

Comment:
Change tests prefix from `testCreateAssessorAdviceNarration` to `testCreateAssessorAdviceNarrationParam`